package.json

```diff
@@ -29,7 +29,7 @@
     "doc": "jsdoc -c .jsdocrc",
     "sia-repo": "go get -u github.com/NebulousLabs/Sia/...",
     "sia-import": "(test -d Sia || mkdir -p Sia) && (cp $GOPATH/bin/siad Sia & cp $GOPATH/bin/siac Sia)",
-    "sia-kill": "killall siad; while pgrep siad; do sleep .5; done;",
+    "sia-kill": "killall -s SIGKILL siad; while pgrep siad; do sleep .5; done;",
```

Comment:
Don't send `SIGKILL`. `SIGTERM` gives siad a chance to shutdown cleanly. Sending `SIGKILL` could cause corruption.